OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
Consider it the first real test of your balance council :wink:

The effect of this can't easily be quantified any deeper than what I outlined above, so testing will need to be done on bleed after this and the footprint changes have been merged.